pages/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import Image from 'next/image';
+import Link from 'next/link';
+import styles from './navbar.module.scss';
+
+const Navbar = () => {
+  const GITHUB_LOGO = "/icons/github.png"

```

Comment:
can we use`import` statements ?

Reply:
No, I removed-> import styles from './navbar.module.scss';